dirty_cat/super_vectorizer.py

```diff
@@ -250,20 +276,35 @@ def _clone_transformers(self):
             self.low_card_cat_transformer_ = clone(self.low_card_cat_transformer)
         elif self.low_card_cat_transformer is None:
             self.low_card_cat_transformer_ = OneHotEncoder()
+        elif self.low_card_cat_transformer == "remainder":
+            self.low_card_cat_transformer_ = self.remainder
         else:
             self.low_card_cat_transformer_ = self.low_card_cat_transformer
 
         if isinstance(self.high_card_cat_transformer, sklearn.base.TransformerMixin):
             self.high_card_cat_transformer_ = clone(self.high_card_cat_transformer)
         elif self.high_card_cat_transformer is None:
             self.high_card_cat_transformer_ = GapEncoder(n_components=30)
+        elif self.high_card_cat_transformer == "remainder":
+            self.high_card_cat_transformer_ = self.remainder
         else:
             self.high_card_cat_transformer_ = self.high_card_cat_transformer
 
+        if isinstance(self.numerical_transformer, sklearn.base.TransformerMixin):
+            self.numerical_transformer_ = clone(self.numerical_transformer)
+        elif self.numerical_transformer is None:
+            self.numerical_transformer_ = "passthrough"
+        elif self.numerical_transformer == "remainder":
+            self.numerical_transformer_ = self.remainder
+        else:
```

Comment:
I'm not sure that I am following the logic fully. If I understand things right, if I put numerical_transformer="pasthrough" (a typo on "passthrough", the numerical_transformer ends up being the high_card_cat_transformer. This sounds like a dangerous behavior, as it will lead to people having bugs in their pipelines.

Reply:
Ah, no, sorry that was en error, it should be `self.numerical_transformer_ = self.numerical_transformer`

Reply:
And so if there was a typo, it will crash down the line when the ColumnTransformer's `fit_transform` will call `_validate_transformers`.

Reply:
OK, thanks. Please add a comment that says that.